docs/release-notes-v17.0.md

```diff
@@ -0,0 +1,89 @@
+# Percona Distribution for PostgreSQL 17.0 ({{date.17_0}})
+
+[Installation](installing.md){.md-button}
+[Upgrade](major-upgrade.md){.md-button}
+ 
+
+We are pleased to announce the launch of Percona Distribution for PostgreSQL 17.0 - a solution with the collection of tools from PostgreSQL community that are tested to work together and serve to assist you in deploying and managing PostgreSQL. The aim of Percona Distribution for PostgreSQL is to address the operational issues like High-Availability, Disaster Recovery, Security, Observability, Spatial data handling, Performance and Scalability and others that enterprises are facing.
+
+This release of Percona Distribution for PostgreSQL is based on **Percona Server for PostgreSQL 17** - a binary compatible drop in replacement of [PostgreSQL 17 :octicons-link-external-16:](https://www.postgresql.org/docs/17/release-17.html). Percona Server for PostgreSQL 17 extends the Storage Manager API to hook in custom storage managers and introduce the encryption of indexes (experimental feature) as part of the Transparent Data Encryption (TDE) solution. 
+
+Both Percona Server for PostgreSQL 17 and PostgreSQL Community 17 function identically enabling you to migrate from one to another. However, index-level encryption is available only with Percona Server for PostgreSQL. So if you encrypted indexes, you cannot switch to the upstream PostgreSQL without losing this data.
+
+## Release Highlights
+
+Percona Distribution for PostgreSQL 17 features a lot of new functionalities and enhancements to performance, replication, monitoring, developer experience and more. Among them are the following:
```

Comment:
I think we should give what's due to the upstream here pointing that these come with PostgreSQL 17 comes packed with these improvements and that Percona Distribution gets all the same benefits plus the extension of the API from Percona

Reply:
Updated